src/tree-node/index.js

```diff
@@ -61,6 +61,20 @@ class TreeNode extends PureComponent {
     readOnly: PropTypes.bool
   }
 
+  getAriaAttributes = () => {
+    const { _children, _depth, checked, disabled, expanded, readOnly, simpleSelect } = this.props
+    const attributes = {}
+
+    attributes.role = simpleSelect ? 'option' : 'treeitem'
+    attributes['aria-selected'] = checked
+    attributes['aria-level'] = _depth || 0 + 1
+    attributes['aria-expanded'] = _children && expanded
+    if (simpleSelect) {
```

Comment:
A tree dropdown can have disabled/readonly nodes as well. Instead this should be negated around level and expanded which is not applicable on a flat dropdown.